apps/user/services/authentication.go

```diff
@@ -115,3 +141,38 @@ func (service AuthenticationService) Logout(c *gin.Context) int {
 	}
 	return http.StatusUnauthorized
 }
+
+// getAccountEmail ...
+func getAccountEmail(person *plus.Person) string {
```

Comment:
this method should return error in case of no account email.

Reply:
Actually, there will be no case where account email is missing. This description of Person.Emails
```
	// Emails: A list of email addresses that this person has, including
	// their Google account email address, and the public verified email
	// addresses on their Google+ profile. The plus.profile.emails.read
	// scope is needed to retrieve these email addresses, or the email scope
	// can be used to retrieve just the Google account email address.
```

Reply:
I think code should still handle the case. As of now its returning "" and that the caller is not checking for it.

Reply:
As discussed, this is handled in code. Added error log to show if code ever reached here as ideally it should not